dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/OracleEvent.scala

```diff
@@ -208,6 +226,20 @@ object OracleEvent {
 
         val attestations = sortedEventDbs.flatMap(_.attestationOpt)
 
+        val outcomes = decomp match {
+          case _: SignedDigitDecompositionEventDescriptor =>
+            val sign = DigitDecompositionSignAttestation(
+              sortedEventDbs.head.outcomeOpt.get == "+")
+            val digits = sortedEventDbs.tail.map { eventDb =>
+              DigitDecompositionAttestation(eventDb.outcomeOpt.get.toInt)
```

Comment:
Why is it safe to access `eventDb.outcomeOpt.get`? I don't see where we are guranteed by the type system that this will always be defined

Reply:
This is a signed/completed event, so it should have the outcome set